implementations/rust/ockam/ockam_node/src/router/start_processor.rs

```diff
@@ -34,6 +34,7 @@ async fn start(
         addr.clone().into(),
         msgs,
         ctrl,
+        Arc::new(0.into()), // don't track metrics
```

Comment:
Why not?

Reply:
Because currently processors have to meaningful parameters to keep track of (for example incoming message queue)

Reply:
Processors have mailbox, not sure we use it though